src/redux/sagas/order.saga.js

```diff
@@ -71,8 +71,7 @@ function* orderSaga() {
   yield takeLatest('FETCH_CUSTOMER_ORDERS', fetchCustomerOrders);
   yield takeLatest('ADD_SHIPPING_INFO', addShipping);
   yield takeLatest('ADD_SAMPLE_INFO', addSampleInfo);
-  yield takeLatest('INITIAL_SAMPLE_ORDER', initialSampleOrder )
-  yield takeLatest('ADD_ORDER', addOrder);
```

Comment:
I think the add order is needed for adding from the sample page. unless somebody changed that dispatch call